tests/data/checkpoint/checkpoint_for_testing/model_config.yaml

```diff
@@ -0,0 +1,144 @@
+modalities_setup:
+  run_mode: FROM_SCRATCH
+  settings:
+    global_num_seen_samples: 0
+
+wandb:
+  project_name: modalities
+  mode: ONLINE
+
+data:
+  sample_key: "input_ids"
+  target_key: "target_ids"
+  sequence_len: 128
+  train_dataloader:
+    type_hint: LLMDataLoader
+    config:
+      dataloader_tag: "train"
+      num_workers: 2
+      pin_memory: true
+      shuffle: false
+      batch_sampler:
+        type_hint: BatchSampler
+        config:
+          batch_size: 8 # per rank
+          drop_last: false
+          sampler:
+            type_hint: DistributedSampler
+            config:
+              rank: ${training.global_rank}
+              num_replicas: ${training.world_size}
+              shuffle: true
+      dataset:
+        type_hint: PackedMemMapDatasetContinuous
+        config: 
+          raw_data_path: /home/richard-rutmann/s3/projects/remote_projects/modalities/data/sample_datasets/redpajama_v2/mem_map/redpajama_v2_samples_1024_train.pbin
```

Comment:
@rrutmann could you check if the tests run for you, if you change this to relative paths?

Reply:
Done in 595e007bea194fb4cad04b09502464c50f79222a